helix-term/src/ui/prompt.rs

```diff
@@ -59,10 +59,11 @@ impl Prompt {
         history_register: Option<char>,
         mut completion_fn: impl FnMut(&str) -> Vec<Completion> + 'static,
         callback_fn: impl FnMut(&mut Context, &str, PromptEvent) + 'static,
+        line: Option<String>,
```

Comment:
Given that most prompts set this to None I'd get rid of the parameter and rather manually set `prompt.line =` where you need it (it's already marked as pub)

Reply:
Alright, will refactor.